modules/dynamic-apiserver/pkg/app/app.go

```diff
@@ -175,56 +178,180 @@ func (srv *Service) WaitForServiceBackup(ctx context.Context, serviceId *string,
 	return errors.New("Retries exceeded, backup is not ready")
 }
 
+func (srv *Service) WaitForService(ctx context.Context, serviceId *string, maxRetries int) error {
+	timeout := time.Second
+	for i := maxRetries; i > 0; i-- {
+
+		result := new(kuberlogiccomv1alpha1.KuberLogicService)
+		err := srv.kuberlogicClient.Get().
+			Resource(serviceK8sResource).
+			Name(*serviceId).
+			Do(ctx).
+			Into(result)
+		if err != nil {
```

Comment:
We might have a transient issue here so just continue the loop

Reply:
I'm not sure that we need to continue in case if service not found or in case error of k8s apiserver.